packages/app-mobile/components/screens/ConfigScreen.tsx

```diff
@@ -320,6 +323,35 @@ class ConfigScreenComponent extends BaseScreenComponent {
 		return 0;
 	}
 
+	private handleBackButtonPress = (): boolean => {
+		const goBack = async () => {
+			BackButtonService.removeHandler(this.handleBackButtonPress);
+			BackButtonService.back();
+		};
+
+		if (this.state.changedSettingKeys.length > 0) {
+			Alert.alert(
+				_('Save changes?'),
```

Comment:
Does the dialog look ok without the title? I don't think we need it because it's "Save changes?" => "There are unsaved changes" => "Save changes", so too many repetitions.